docs/portable-modules/System.Management.Automation-lib.cs

```diff
@@ -0,0 +1,11158 @@
+    public override object ConvertTo(System.Management.Automation.PSObject sourceValue, System.Type destinationType, System.IFormatProvider formatProvider, bool ignoreCase) { return default(object); }
+    public override object ConvertTo(object sourceValue, System.Type destinationType, System.IFormatProvider formatProvider, bool ignoreCase) { return default(object); }
+    public static System.Guid GetFormatViewDefinitionInstanceId(System.Management.Automation.PSObject instance) { return default(System.Guid); }
+    public static uint GetParameterSetMetadataFlags(System.Management.Automation.PSObject instance) { return default(uint); }
+  }
+  public enum ExecutionPolicy {
+    // Fields
+    AllSigned = 2,
+    Bypass = 4,
+    Default = 3,
+    RemoteSigned = 1,
+    Restricted = 3,
+    Undefined = 5,
+    Unrestricted = 0,
+  }
+  public enum ExecutionPolicyScope {
+    // Fields
+    CurrentUser = 1,
+    LocalMachine = 2,
+    MachinePolicy = 4,
+    Process = 0,
+    UserPolicy = 3,
+  }
+  public sealed partial class PSAuthorizationManager : System.Management.Automation.AuthorizationManager {
+    // Constructors
+    public PSAuthorizationManager(string shellId) : base (default(string)) { }
+     
+    // Methods
+    protected internal override bool ShouldRun(System.Management.Automation.CommandInfo commandInfo, System.Management.Automation.CommandOrigin origin, System.Management.Automation.Host.PSHost host, out System.Exception reason) { reason = default(System.Exception); return default(bool); }
+  }
+#if SNAPINS
+  [System.ComponentModel.RunInstallerAttribute(true)]
+  public sealed partial class PSCorePSSnapIn : System.Management.Automation.PSSnapIn {
+    // Constructors
+    public PSCorePSSnapIn() { }
+     
+    // Properties
+    public override string Description { get { return default(string); } }
+    public override string DescriptionResource { get { return default(string); } }
+    public override string[] Formats { get { return default(string[]); } }
+    public override string Name { get { return default(string); } }
+    public override string[] Types { get { return default(string[]); } }
+    public override string Vendor { get { return default(string); } }
+    public override string VendorResource { get { return default(string); } }
+     
+    // Methods
+  }
+#endif
+  public static partial class ToStringCodeMethods {
+    // Methods
+    public static string PropertyValueCollection(System.Management.Automation.PSObject instance) { return default(string); }
+    public static string Type(System.Management.Automation.PSObject instance) { return default(string); }
+    public static string XmlNode(System.Management.Automation.PSObject instance) { return default(string); }
+    public static string XmlNodeList(System.Management.Automation.PSObject instance) { return default(string); }
+  }
+}
+namespace Microsoft.PowerShell.Cim {
```

Comment:
Caveat: we want the first iteration of this to be the most minimal surface area. At a point in the future, if we were to add CIM support, we could re-add this to a later version of this API.